examples/select-with-defaults-and-timers.flix

```diff
@@ -27,8 +27,8 @@ def recvWithTimeout(c: Receiver[Int32, Static]): Int32 \ IO =
 /// Sends values on both after one minute.
 /// Receives from both using defaults and timeouts.
 def main(): Unit \ IO = {
-    let (s1, r1) = Channel.buffered(1);
-    let (s2, r2) = Channel.buffered(1);
+    let (s1, r1) = Channel.buffered(Static, 1);
+    let (s2, r2) = Channel.buffered(Static, 1);
```

Comment:
Is the plan to use Static now and then go back in the PR and introduce regions?

Reply:
I stuck with Static here and elsewhere in an attempt to keep this (already very large!) PR as small as possible. I was planning to revisit when we have worked out what the effect of channel operations should be.